# used for steering and brake dynamixels.
bool health_ok

string error
